lib/timeline/TimeStep.js

```diff
@@ -582,7 +582,7 @@ TimeStep.prototype.getLabelMinor = function(date) {
       if(this.isMajor() && date.weekday() !== 0){
           return "";
       }
-      break;
+      //no break; fall to the next case (default)
```

Comment:
From a code style POV, I wasn't sure what would be preferred 

1) use an else statement in the `week` case, so that it provides the default labeling. e.g.

```
if(this.isMajor() && date.weekday() !== 0){
     return "";
} else {
    return (format && format.length > 0) ? this.moment(date).format(format) : '';
}
```
2) remove the break statement, and let this fall to the default case. (I opted for this, with a  comment)